docs/blog/posts/kubescape-3-cli-improvements/index.md

```diff
@@ -0,0 +1,93 @@
+---
+date: 2023-09-29
+categories:
+  - Announcements
+  - Project
+authors:
+  - vladklokun
+slug: 'kubescape-3-cli-improvements'
+---
+
+# Kubescape 3.0: CLI improvements
+
+In the latest release of Kubescape, we completely overhauled the CLI experience to make it easier and faster for you to improve the security of your clusters.
+
+Watch a short video for a demonstration of the new CLI and its benefits, or read on.
+
+<div class="video-wrapper">
+  <iframe width="640" height="360" src="https://www.youtube.com/embed/ul5Wry6sS8w?si=V8CStezYDZa_4Dtd" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
We updated the video to add a branded closer.
You can find it here: https://drive.google.com/file/d/1SkdFuc29ZjGwzrsMzZ7ekbwAgUVtmNkj/view?usp=sharing